### PR TITLE
Updated Work and Education History for ZThatcher

### PR DESCRIPTION
**Pull request by zthatch**
First pull request to update career and education information in db/people.yml

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -2892,12 +2892,33 @@ zguguchia:
...
 zthatcher:
   active: true
-  aka: []
+  aka: 
+    - Zach Thatcher
+    - Thatcher, Zachary A.
+    - Thatcher, Z. A. 
+    - Zachary A. Thatcher
   avatar: https://github.com/zthatch.png
   bio: bio
```
Comment (sbillinge) [inline db/people.yml:2901]: If possible please put a few words here about yourself

File: db/people.yml [hunk 2]
```diff
@@ -2892,12 +2892,33 @@ zguguchia:
...
+      end_year: 2017
+      end_month: 3
+    - begin_year: 2020
+      begin_month: 8
+      degree: PhD in Material Science
+      institution: Columbia University
+      group: bg
   email: zachary.alan.thatcher@gmail.com
   employment:
+    - being_year: 2017
```
Comment (sbillinge) [inline db/people.yml:2916]: this should be reverse chronological so pls switch order

**Comment by zthatch** [comment]
> Thanks @zthatch this is great. I left a couple of comments. It is not passing CI because of some validation errors (click on "details" in the travis CI section to see why it is failing). You can run validation on the database using regolith validate. Please speak to Berrak to get that going.

Will do! I thought that I saw the error, but must be misinterpreting the error log.

**Comment by sbillinge** [comment]
Super!  Thanks so much

**Pull request merged**